src/server/util/requestValidation.js

```diff
@@ -1,9 +1,9 @@
 const { apiError } = require('./api');
 
 const validateRequestUsernameAndEmail = function(req, res, next) {
-    const { username, email } = req.body;
+    const { username, email, user } = req.body;
 
-    if (!username && !email) {
+    if (!username && !email && !user) {
```

Comment:
Nonblocking: I see how you're using a user object, and it seems clear. I'm wondering if the verbiage is a little confusing? Perhaps there should be a comment about the difference between the user object vs the username field?

Reply:
Good call, I'll add a comment to explain more what is expected for `user`.